src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -543,7 +543,7 @@ public function formAction($id, Request $request)
         }
 
         $doctrine = $this->getDoctrine()->getManager();
-        $attributeGroups = $doctrine->getRepository('PrestaShopBundle:Attribute')->findByLangAndShop(1, 1);
+        $attributeGroups = $doctrine->getRepository('PrestaShopBundle:Attribute')->findByLangAndShop((int) $languages[0]['id_lang'], (int) ['shop_id']);
```

Comment:
You must make sure `$languages` has `[0]` and `['shop_id']` isn't an id. 
You can do:
```php
$language = empty($languages[0]) ? ['id_lang' => 1, 'shop_id' => 1] ? $language[0];
$attributeGroups = $doctrine->getRepository('PrestaShopBundle:Attribute')->findByLangAndShop(
    (int) $language['id_lang'], 
    (int) $language['shop_id']
);
```

@Quetzacoalt91 @matks wdyt?

Reply:
Yes, I agree.

Reply:
I think it should be like: 
$language = empty($languages[0]) ? ['id_lang' => 1, 'shop_id' => 1] : $language[0];
$attributeGroups = $doctrine->getRepository('PrestaShopBundle:Attribute')->findByLangAndShop(
    (int) $language['id_lang'], 
    (int) $language['shop_id']
);

Reply:
`(int) ['shop_id']` isn't an id..